app/presenters/image_upload_response_presenter.rb

```diff
@@ -78,6 +78,7 @@ def attention_with_barcode?
   end
 
   def ocr_pii
```

Comment:
I feel like I should probably know this...but what is OCR?

I tried doing a quick search but came across references that seemed to range from a concept to a specific org, so I wanted to clarify.

Reply:
@night-jellyfish , I think OCR here means optical character recognization?